.github/workflows/submit-gradle-dependencies.yml

```diff
@@ -1,9 +1,5 @@
 name: Submit dependencies to GitHub Dependency Graph
-on:
-  push:
-    branches:
-      - trunk
-      - release/*
+on: [push, pull_request]
```

Comment:
I expected that with this change we might get 
> This pull request would resolve a Dependabot alert on <dependency>

message like e.g. here: https://github.com/wordpress-mobile/WordPress-Android/pull/19492

but it seems it doesn't work like this. Reverting this change.

Reply:
I've asked a question here: https://github.com/gradle/github-dependency-graph-gradle-plugin/issues/154

Reply:
Someone actually responded to you on that! 👍